class/nfs-subdir-external-provisioner.yml

```
@@ -10,12 +10,12 @@ parameters:
       - input_paths:
           - nfs-subdir-external-provisioner/component/app.jsonnet
         input_type: jsonnet
-        output_path: apps/
+        output_path: apps/${_instance}
       - input_paths:
           - nfs-subdir-external-provisioner/component/main.jsonnet
         input_type: jsonnet
-        output_path: nfs-subdir-external-provisioner/
-      - output_path: nfs-subdir-external-provisioner/01_helmchart
+        output_path: nfs-subdir-external-provisioner/${_instance}
```

Since all instances are currently deployed into the same namespace (which is totally fine), you don't need to create the namespace object for each instance. So here it's sufficient to just have the original output path:

```suggestion
        output_path: nfs-subdir-external-provisioner/
```

---

Removed the distinction for the namespace, now only helmcharts are put into component-specific directories